java/se/sics/mspsim/util/DotDiagram.java

```diff
@@ -359,7 +359,7 @@ protected void paintComponent(Graphics g0) {
       sizeX = width - 2;
 
       if (totMax < 0) {
-        factor = (sizeY - 15) / (double) (0 - totMin);
+        factor = (sizeY - 15) / (double) (-totMin);
```

Comment:
Perhaps also remove the parenthesis as they are no longer needed? Same for line 380?

Reply:
Fixed.